src/transport/concentration_model.cc

```diff
@@ -123,26 +123,12 @@ ConcentrationTransportModel::ModelEqData::ModelEqData()
 
 
 
-UnitSI ConcentrationTransportModel::balance_units()
```

Comment:
TransportDG<ConcentrationTransportModel> obtains the instance of Balance from TransportOperatorSplitting, so here it does not need to define its units.